index.html

```diff
@@ -19,7 +19,25 @@
   
      
   <div class="container home">
-   
+    <header>
```

Comment:
```suggestion
    <header>
        <div>
                  <h1>Fun Bus</h1>
                  <nav>
                      <a href="#">Home</a>
                      <a href="#">About Us</a>
                      <a href="#">Blog</a>
                      <a href="#">Contact</a>
                  </nav>
        </div>
    </header>
```
You could add a div to the nav and move it our of the container to make the border go across the screen